IncBackups/IncBackupsControl.py

```diff
@@ -184,8 +185,8 @@ def awsFunction(self, fType, backupPath=None, snapshotID=None, bType=None, delet
 
                     key, secret = self.getAWSData()
 
-                    command = 'export AWS_ACCESS_KEY_ID=%s AWS_SECRET_ACCESS_KEY=%s  && restic -r s3:s3.amazonaws.com/%s forget %s --password-file %s' % (
-                        key, secret, self.website, snapshotID, self.passwordFile)
+                    command = 'export AWS_ACCESS_KEY_ID=%s AWS_SECRET_ACCESS_KEY=%s  && restic -r s3:s3.amazonaws.com/%s forget %s --prune --password-file %s' % (
```

Comment:
combined the forget and prune mechanisms, and changed it so all jobids are forgotten at once (e.g. `restic -r remotename forget 2385238d 329349s9 23458s3j --prune --password-file... `) and a single prune is done afterwards - the saves considerable time because prune is VERY expensive